android/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 
   <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-      package="nl.kevinboosten.keepawake.capacitorkeepawake">
+      package="com.getcapacitor.community.keepawake.keepawake">
```

Comment:
You should be able to just do `com.getcapacitor.community.keepawake`. I think the plugin template generates this incorrect (it has already been fixed)

Reply:
Changed it!